tests/api2/activedirectory.py

```diff
@@ -257,31 +243,31 @@ def test_19_setting_up_smb(request):
 
 
 @pytest.mark.parametrize('data', ["description", "guest"])
-def test_20_verify_the_value_of_put_smb_object_value_of_(request, data):
+def test_18_verify_the_value_of_put_smb_object_value_of_(request, data):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     assert results.json()[data] == payload[data], results.text
 
 
-def test_21_get_smb_data(request):
+def test_19_get_smb_data(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     global results
     results = GET("/smb/")
     assert results.status_code == 200, results.text
 
 
 @pytest.mark.parametrize('data', ["description", "guest"])
-def test_22_verify_the_value_of_get_smb_object_(request, data):
+def test_20_verify_the_value_of_get_smb_object_(request, data):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     assert results.json()[data] == payload[data], results.text
 
 
-def test_23_creating_a_smb_share_on_smb_path(request):
+def test_21_creating_a_smb_share_on_smb_path(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     global payload, results, smb_id
     payload = {
         "comment": "My Test SMB Share",
-        "path": SMB_PATH,
-        "name": SMB_NAME,
+        "path": smb_path,
+        "name": smb_name,
         "guestok": True,
```

Comment:
Probably need to clearly differentiate between tests where we're explicitly testing guest access and not testing guest access. Creating a share with `guestok == True` alters global SMB server configuration. Maybe start with False, then update the share immediately prior to testing guest access in AD environment.

Reply:
actually that should be remove or False thanks